doc-requirements.txt

```diff
@@ -0,0 +1,5 @@
+pip>=9.0.0
+sphinx>=1.4
+setuptools>=23.0.0
```

Comment:
We might want to make ths `>=31.0.1`, given the namespace-related issues in some of the versions since this one.